variables.tf

```diff
@@ -141,3 +141,45 @@ variable "nat_timeout" {
   type        = string
   default     = "10m"
 }
+
+variable "external_ips_global" {
+  description = <<-EOT
+  A list of GLOBAL external IPs to be created that can be used for external load-balancers, GKE
+  ingress IPs, etc. See description of each expected field.
+
+  name (MANDATORY): The custom portion for generating a formatted name of the external IP. This
+  field is also used by terraform as the output key/alias.
+
+  bwc_fullname (OPTIONAL): The backward-compatible-fullname (if provided) will be used for naming
```

Comment:
`backward_compatible_name` sounds like a better naming.

Reply:
Hmm. Was looking for a way to shorten the long name. 👍🏽